README.md

```diff
@@ -8,7 +8,7 @@ Documentation is starting to be built in the [wiki](https://github.com/aced-diff
 This package is currently focused on bulk systems. For organic molecules, [MolecularGraph](https://github.com/mojaie/MolecularGraph.jl) is recommended.
 PubChem stores many molecular features for the compounds they catalog, and their data can be accessed via [PubChemCrawler](https://github.com/JuliaHealth/PubChemCrawler.jl).
 
-## Features
+## FeatureDescriptors
```

Comment:
hahaha my guess is this might have been the result of a universal replace, but I intended this header to just be a description of features of the package, but hadn't gotten around to populating it yet 🤪

Reply:
Resolved in 4b0abda.